ibm/service/transitgateway/data_source_ibm_tg_connection_prefix_filters.go

```diff
@@ -0,0 +1,138 @@
+// Copyright IBM Corp. 2017, 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package transitgateway
+
+import (
+	"fmt"
+	"github.com/IBM/networking-go-sdk/transitgatewayapisv1"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"time"
+)
+
+const (
+	tgPrefixFilters = "prefix_filters"
+	tgAction        = "action"
+	tgBefore        = "before"
+	tgGe            = "ge"
```

Comment:
can we have user friendly names for ge and le?
what does api and cli use?

Reply:
The API/cli/customer docs all use "ge" and "le", so I would prefer to keep those parameters consistent with them. I did update the resource and data docs to explain more.